src/main/java/io/appform/dropwizard/actors/base/utils/NamingUtils.java

```diff
@@ -1,16 +1,28 @@
 package io.appform.dropwizard.actors.base.utils;
 
+import io.appform.dropwizard.actors.utils.CommonUtils;
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class NamingUtils {
 
+    public static final String FEATURE_ENV_NAME = "feature_env_name";
+
     public static String queueName(String prefix, String name) {
-        return String.format("%s.%s", prefix, name);
+        String namespaced = prefixWithNamespace(name);
+        return String.format("%s.%s", prefix, namespaced);
     }
 
     public static String sanitizeMetricName(String metric) {
         return metric == null ? null : metric.replaceAll("[^A-Za-z\\-0-9]", "").toLowerCase();
     }
+
+    public static String prefixWithNamespace(String name) {
```

Comment:
`val`

Reply:
@santanusinha Changed the local variables to final. Did you meant the same for function argument as well?